quesma/v2/core/quemsa_logger.go

```diff
@@ -0,0 +1,146 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
```

Comment:
Typo in file name, should be `quesma_logger.go` instead of `quemsa_logger.go`

Reply:
Shame on me. Fixed.